tests/test_learning_path.py

```diff
@@ -0,0 +1,66 @@
+from collections import OrderedDict
+import domain.tutoringModel.generate_learning_path as LP
+import pytest
+
+generate_learning_path = LP.GenerateLearningPath()
+
+
+def test_get_learning_path_correct_result():
+    input_learning_style = {"AKT": 5, "INT": 9, "VIS": 9, "GLO": 9}
+    # ZF-ÜB-#AN-#SE-RQ-AB-#ZL-#BE-FO
+    learning_path = {
+        'ZF': 99, 'ÜB': 14, 'SE': 5, 'AN': 5,
+        'RQ': 4, 'AB': 0, 'ZL': -5, 'BE': -5, 'FO': -13}
+    assert generate_learning_path.get_learning_path(
+        input_learning_style) == learning_path
+
+
+def test_get_learning_path_correct_result_2():
+    input_learning_style = {"REF": 1, "SNS": 7, "VIS": 5, "SEG": 5}
+    # result of Learning Path : AN-BE-AB-#SE-#ÜB-RQ-ZF-FO-ZL
+    learning_path = {
+        'AN': 11,  'BE': 8, 'AB': 7, 'SE': 6, 'ÜB': 6,
+        'RQ': 1, 'ZF': 0, 'FO': -6, 'ZL': -11}
+
+    assert generate_learning_path.get_learning_path(
+        input_learning_style) == learning_path
+
+
+def test_get_learning_path_correct_result_3():
+    # 3- Test
+    input_learning_style = {"AKT": 3, "SNS": 7, "VIS": 3, "GLO": 3}
+
+    # result of Learning Path : ZF-AN-#SE-#ÜB-#AB-BE-FO-RQ-ZL
+    learning_path = {'ZF': 99, 'AN': 13, 'SE': 10, 'ÜB': 10,
+                     'AB': 10, 'BE': 7, 'FO': 0, 'RQ': -3, 'ZL': -13}
+    assert generate_learning_path.get_learning_path(
+        input_learning_style) == learning_path
+
+
+def test_get_learning_path_correct_result_4():
+    # 4- Test
```

Comment:
As many comments as necessary, as few as possible

Reply:
I ll remove this